RoboteQ double channel launch file

nabilgabbouj opened the pull request:
Adds support for two channel RoboteQ drivers. Usage : roslaunch roboteq_driver roboteq.launch dual:=true
To access encoders and feedback for each channel check topics : /left/feedback or /right/feedback

nabilgabbouj committed:
RoboteQ double channel launch file

Adds support for two channel RoboteQ drivers. Usage : roslaunch roboteq_driver roboteq.launch dual:=true
Feedback namespaces : Left and Right.

mikepurvis commented:
Can you change the existing `example.launch` file rather than adding another one, please?

nabilgabbouj committed:
Delete roboteq.launch

nabilgabbouj committed:
Double channel drivers support

Usage roslaunch roboteq_driver example.launch dual:=true
Feedback topics: /left/feedback and /right/feedback
